internal/gatewayapi/translator.go

```diff
@@ -839,8 +839,8 @@ func (t *Translator) ProcessHTTPRoutes(httpRoutes []*v1beta1.HTTPRoute, gateways
 
 								if !canAddHeader {
 									parentRef.SetCondition(
-										v1beta1.RouteConditionResolvedRefs,
-										metav1.ConditionFalse,
+										v1beta1.RouteConditionAccepted,
+										metav1.ConditionTrue,
 										v1beta1.RouteReasonUnsupportedValue,
 										fmt.Sprintf("RequestHeaderModifier Filter already configures request header: %s to be added, ignoring second entry", headerKey),
 									)
```

Comment:
From my read of the spec, we should either be ignoring the duplicate header, or appending its value:

- https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPHeaderFilter
- https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPHeader

But it doesn't seem like we need to be setting a condition here.

Regardless, the combo of `Accepted: true` with reason `UnsupportedValue` does not seem valid -- if we're using the `UnsupportedValue` reason, then the status should be `False`.

Reply:
/cc @AliceProxy

Reply:
> From my read of the spec, we should either be ignoring the duplicate header, or appending its value:

The code currently ignores duplicate headers from the add/remove operations. Commit e14991a no longer sets a status condition for these cases and therefore an `Accepted=true` condition is surfaced.